projects/portal/src/app/views/mint/cdps/cdps.component.html

```diff
@@ -2,7 +2,7 @@
   <div class="flex flex-row">
     <h3 matSubheader>CDPs</h3>
     <span class="flex-auto"></span>
-    <button mat-flat-button routerLink="/mint/cdps/create" color="accent">
+    <button mat-flat-button routerLink="/cdp/cdps/create" color="accent">
```

Comment:
As you will see when you check the operation, I have changed it from /cdp/cdps to /mint/cdps.
It's about time they recognize it.

Reply:
Ohh, sorry.
I will check changed files by myself from now on.

Reply:
Ohh, sorry.
I will check changed files by myself from now on.